machines/vc.ts

```diff
@@ -199,7 +200,13 @@ export const vcMachine =
         },
 
         setVcUpdate: (context, event) => {
-          context.vcs[VC_ITEM_STORE_KEY(event.vc)] = event.vc;
+          Object.keys(context.vcs).map((vc) => {
+            if (isVcKeyMatch(vc, VC_ITEM_STORE_KEY(event.vc))) {
+              context.vcs[VC_ITEM_STORE_KEY(event.vc)] = context.vcs[vc];
```

Comment:
can directly refer vc instead of context.vcs[vc] as vc is available in map

Reply:
here vc is the key available in context.vc. I was assigning the value of vc to new key to get the vc value we are using `context.vcs[vc]` since context.vcs is an `object` it will return the value of vc in context.vcs